### PR TITLE
Change Tracking interface Javadocs

### PR DESCRIPTION
**Pull request by Alec-Shay**
/cc @Preston-Crary

Javadocs for the new change tracking API, to go together with: [LPS-102383](https://issues.liferay.com/browse/LPS-102383)

I also included a commit to update the reviewer information in the Javadoc submission process article, since that was out of date.

Please let us know if there are any issues with this. Thank you.

### CODE REVIEW

**Comment by Alec-Shay** [comment]
ci:reopen

File: modules/apps/change-tracking/change-tracking-spi/src/main/java/com/liferay/change/tracking/spi/resolver/context/ConstraintResolverContext.java [hunk 1]
```diff
@@ -42,36 +42,34 @@
 		throws E;
 
 	/**
-	 * Returns the source CT model has a unique constraint conflict with the
-	 * target CT model.
+	 * Returns the source change tracking model has a unique constraint conflict with the
```
Comment (Preston-Crary) [inline modules/apps/change-tracking/change-tracking-spi/src/main/java/com/liferay/change/tracking/spi/resolver/context/ConstraintResolverContext.java]: Missing "that". Reviewed all your changes and this is the only problem I saw.

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/627de159e1...c7dd0c0497

**Comment by brianchandotcom** [comment]
@Preston-Crary @Alec-Shay can you readd with ```@review``` so the docs team knows to edit them. And please use one LPS / COMMERCE ticket that I can tie this to.

Type:

git grep @review **.java

**Comment by brianchandotcom** [comment]
@Preston-Crary @Alec-Shay ,sorry, type:

```
git grep @review **.java
```

**Comment by Alec-Shay** [comment]
@brianchandotcom I'm creating an LPS and getting ready to resend.

To clarify, I need to add `@review` to all these Javadocs for further edits from someone else? I thought my review constituted those edits, unless there is another team apart from KM this will need to go through. I could be mistaken.

**Comment by Alec-Shay** [comment]
Created an LPS and re-sent: https://github.com/brianchandotcom/liferay-portal/pull/94229

**Pull request closed without merging**